### ISSUE TITLE
Have some trouble with typeerror

### ISSUE THREAD

**Issue by cnleo** [evansiroky/timezone-boundary-builder#136]
Hey, I am try this nice tool but run in a "simple" error. I am not familiar with the code and hope someone with more experience will found it easy – maybe. Or can give me the right direction. The `combined-osm-zones.json` in the `working` directory is created but empty and I am not sure about why?  Chicago and New York created and well writen in `working` and `downloads` folder. Hope someone with more insider insight can help me. 
Thank you in advance.

```
C:\timezone-boundary-builder-master>node --max-old
-space-size=8192 index.js --included_zones America/New_York America/Chicago
Creating downloads dir (C:\timezone-boundary-build
er-master\downloads); Overall progress: 0% done
Creating working dir (C:\timezone-boundary-builder
-master\working); Overall progress: 4.8% done
Creating dist dir (C:\timezone-boundary-builder-ma
ster\dist); Overall progress: 9.5% done
Writing timezone names to file; Overall progress: 14.3% done
Downloading OSM boundaries; Overall progress: 19% done
getting data for America-New_York-tz; Downloading progress: 0% done - ? left
downloading from overpass
Downloading last release for analysis; Overall progress: 23.8% done
getting data for America-Chicago-tz; Downloading progress: 50% done - 0.0 second
s left
downloading from overpass
Downloading OSM TZ boundaries; Overall progress: 28.6% done
getting data for America-New_York-tz; Downloading OSM Zone progress: 0% done - ?
 left
downloading from overpass
Creating timezone boundaries; Overall progress: 33.3% done
makeTimezoneBoundary for America/New_York; Building progress: 0% done - ? left
- init America-New_York-tz
- union undefined
- union undefined
makeTimezoneBoundary for America/Chicago; Building progress: 50% done - 1.2 seco
nds left
- init America-Chicago-tz
- union undefined
- difference undefined
- difference undefined
getting data for America-Chicago-tz; Downloading OSM Zone progress: 50% done - 2
.1 seconds left
downloading from overpass
Merging osm zones; Overall progress: 38.1% done
C:\timezone-boundary-builder-master\index.js:828
    const tzBoundayName = `${tzId.replaceAll('/', '-')}-tz`
                                  ^

TypeError: tzId.replaceAll is not a function
    at C:\timezone-boundary-builder-master\index.j
s:828:35
    at Array.forEach (<anonymous>)
    at combineAndWriteOSMZones (C:\timezone-bounda
ry-builder-master\index.js:827:24)
    at autoScript.mergeOSMZones (C:\timezone-bound
ary-builder-master\index.js:1047:7)
    at runTask (C:\timezone-boundary-builder-maste
r\node_modules\async\dist\async.js:1153:17)
    at C:\timezone-boundary-builder-master\node_mo
dules\async\dist\async.js:1091:35
    at processQueue (C:\timezone-boundary-builder-
master\node_modules\async\dist\async.js:1101:17)
    at taskComplete (C:\timezone-boundary-builder-
master\node_modules\async\dist\async.js:1118:13)
    at C:\timezone-boundary-builder-master\node_mo
dules\async\dist\async.js:1146:21
    at C:\timezone-boundary-builder-master\node_mo
dules\async\dist\async.js:327:20

C:\timezone-boundary-builder-master>
```

**Comment by kohenkatz** [comment]
What version of node are you using (`node -v`)? `replaceAll` was implemented in v15.0, so it won't work if you are using an older version.

**Comment by evansiroky** [comment]
Given the lack of activity on this issue, I am closing. Please reopen if this is still a problem.
